src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
```

Comment:
It is if we call it ourself. While spinning up tests the first thing I did was GameRunner.main(null) and quickly ran into this NPE. Given how much null values we have in our code, this is probably safest. 

We do call main methods ourselves, I see risk that in that logic we would pass in null. To some extent I feel like this is similar to marking parameters final. In general good code shouldn't modify parameters, and things will be immutable, so the final marking on all parameters becomes unnecessary or fluff. The reasoning for having it here. The test requires more changes, I wanted to keep this isolated for now so you are not seeing the test.